CODEOWNERS

```diff
@@ -39,19 +39,8 @@
 go.sum @fleetdm/go
 go.mod @fleetdm/go
 /cmd/ @fleetdm/go
-/orbit/ @lucasmrod @getvictor @roperzh @gillespi314
 /server/ @fleetdm/go
-/server/service/handler.go @lucasmrod @getvictor @roperzh @gillespi314
-/server/mdm/ @roperzh @gillespi314 @lucasmrod @georgekarrv
-/server/worker/ @lucasmrod @getvictor @roperzh @gillespi314 @lukeheath
-/server/vulnerabilities/ @lucasmrod @mostlikelee @getvictor @lukeheath
-/server/cron/ @getvictor @lucasmrod @roperzh @mostlikelee @lukeheath
-/ee/fleetd-chrome @lucasmrod @getvictor @RachelElysia
-/ee/vulnerability-dashboard @eashaw
-/ee/cis @sharon-fdm @lucasmrod @RachelElysia @jacobshandling
-/ee/server/calendar @lucasmrod @getvictor @jacobshandling
-/ee/server/service @roperzh @gillespi314 @lucasmrod @getvictor
-/scripts/mdm @roperzh @gillespi314 @jahzielv @dantecatalfamo
+/orbit/ @lucasmrod @roperzh @lukeheath @georgekarrv @sharon-fdm
```

Comment:
Looks good. I'd add a third backend person here in case Roberto or me are OOO and one of us is the author of a PR that touches orbit.

Reply:
Good point, I've added one backup on each product group.